### DISCUSSION TITLE
[Feature] Support of the TerraForm CDK functionality inside the OpenTofu eco system

### DISCUSSION THREAD

**Discussion by ZPascal** [opentofu/opentofu#1149] [Ideas]
Hello Community,

is there anything planned to also support an opentofo-cdk like the [TF CDK](https://github.com/hashicorp/terraform-cdk) to use programming languages [natively](https://developer.hashicorp.com/terraform/tutorials/cdktf/cdktf-build?in=terraform%2Fcdktf&variants=cdk-language%3Apython) to manage resources?

**Comment by ghost** [comment]
Hi @ZPascal you can use cdktf by setting the `TERRAFORM_BINARY_NAME` environment variable to `tofu`, see [this issue](https://github.com/opentofu/opentofu/issues/601). Please let us know if this works for you.

**Reply by ZPascal** [reply]
Hi @janosdebugs, thank you for the quick response. Sounds great, I'll give it a try.
